core/banking-protocol/src/main/java/de/adorsys/opba/core/protocol/domain/entity/BankProfile.java

```diff
@@ -0,0 +1,38 @@
+package de.adorsys.opba.core.protocol.domain.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.Proxy;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Proxy(lazy = false)
```

Comment:
Typically lazy initialisations lead to issues when entity live more than one transaction. Unless it is really needed I prefer to avoid lazy for entities

Reply:
ok